straxen/plugins/peaklet_processing.py

```diff
@@ -316,6 +319,7 @@ def peak_saturation_correction(records, peaks, to_pe,
         peaks[peak_i]['dt'] = dt
 
     strax.sum_waveform(peaks, records, to_pe, peak_list)
+    return peak_list
```

Comment:
maybe makes sense to just do `strax.compute_widths(peaklets, select_peaks_indicies=peak_list)` here together with the sum_wf, also alleviates the need to return this variable?

Reply:
@zhut19 , did you see my suggestion?

Reply:
Oh, sorry, I missed this. I couldn't do this because that `strax.compute_width` is not a numba function.

Reply:
You could use a numba.typed.List, but then you should check the impact on the performance.